sdks/go/pkg/beam/io/xlang/debeziumio/debezium.go

```diff
@@ -0,0 +1,124 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package debeziumio contains cross-language functionality for using Debezium
+// (http://kafka.apache.org/). These transforms only work on runners that
+// support cross-language transforms.
+//
+// Setup
+//
+// Transforms specified here are cross-language transforms implemented in a
+// different SDK (listed below). During pipeline construction, the Go SDK will
+// need to connect to an expansion service containing information on these
+// transforms in their native SDK.
+//
+// To use an expansion service, it must be run as a separate process accessible
+// during pipeline construction. The address of that process must be passed to
+// the transforms in this package.
+//
+// The version of the expansion service should match the version of the Beam SDK
+// being used. For numbered releases of Beam, these expansions services are
+// released to the Maven repository as modules. For development versions of
+// Beam, it is recommended to build and run it from source using Gradle.
+//
+// Current supported SDKs including expansion service modules
+// * Java
+// 	  - Vendored Module: beam-sdks-java-io-debezium-expansion-service
+// 	  - Run via Gradle: ./gradlew :sdks:java:io:debezium:expansion-service:shadowJar
+//						 java -jar <path-to-debezium-jar> <port>
+
+package debeziumio
+
+import (
+	"reflect"
+
+	"github.com/apache/beam/sdks/v2/go/pkg/beam"
+	"github.com/apache/beam/sdks/v2/go/pkg/beam/core/typex"
+)
+
+// DriverClassName is the type for valid and supported Database connectors for Debezium IO.
+type DriverClassName string
+
+const (
+	// MYSQL connector for Debezium
+	MYSQL DriverClassName = "MySQL"
+	// POSTGRESQL connector for Debezium
+	POSTGRESQL = "PostgreSQL"
+	// ORACLE connector for Debezium
+	ORACLE = "Oracle"
+	// DB2 connector for Debezium
+	DB2 = "Db2"
+)
+
+const readURN = "beam:transform:org.apache.beam:debezium_read:v1"
+
+// readFromDebeziumSchema is config schema that matches exactly with the Java's Debezium IO
+// for cross language payload.
```

Comment:
Please include the link to the Java Debezium IO configuration call so it's possible for later people to find, incase this needs updating.

Reply:
Added the Java's class documentation in package documentation itself.